npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fxa-content-server",
-  "version": "1.110.5",
+  "version": "1.111.1",
```

Comment:
The dependencies didn't update, does npm-shrinkwrap.json need to be updated?

Reply:
Interesting, no dependencies got updated and I am fairly certain I didn't run `npm run shrink`. Not sure how this would have gotten updated 😕

Reply:
Probably via npm install